src/pages/Integrations/SplunkSetup/useSplunkSetup.tsx

```diff
@@ -33,26 +33,15 @@ Steps:
 ["ff59b502-da25-4297-bd88-6934ad0e0d63"] <<- Behavior group ID
```

Comment:
Can you please remove the comment (or update it)?

Reply:
Sure, and sorry I forgot to update comments